plugins/strings.js

```diff
@@ -22,7 +22,9 @@ export const datetimeToString = (source, options = defaultDatetimeOptions) => {
         options.inputFormat || defaultDatetimeOptions.inputFormat
     const outputFormat =
         options.outputFormat || defaultDatetimeOptions.outputFormat
-    return dayjs(source, inputFormat).format(outputFormat)
+
+    const offset = new Date().getTimezoneOffset()
+    return dayjs(source, inputFormat).subtract(offset, 'm').format(outputFormat)
```

Comment:
Do we need this change in this PR? I don't see the changes using this plugin method.

Reply:
one should see the change if you change the default timezone of your system.